### ISSUE TITLE
Curious about usages?

### ISSUE THREAD

**Issue by Merith-TK** [CesiumMC/cesium-fabric#1]
I have a Vanilla server that is just a bunch of enhancements to Vanilla, serverside.

what advantages would this mod have over vanilla?

**Comment by yamayaki1** [comment]
I'm really sorry for answering that late, looks like I missed the notification.

Regarding your question, I don't really think that you would see an advantage over Vanilla's storage format. In fact, the only advantage Cesium has, is that it slightly reduces disk space used by a world compared to Vanilla. Most noticeably in amplified worlds.

In its current state, Cesium is more like a "let's see what happens if I do this and that" project for me to try different things. I'm for example not happy with how LMDB files can only grow bigger or stay the same size, even if I were to delete chunks or they become smaller. I'm looking at how I can could potentially fix that... Anyways.

In short, you probably don't want to use Cesium right now. I hope this helps answer your question.

**Comment by Merith-TK** [comment]
I am mainly looking for ways to potentially enhance server backups, and it looks like cesium uses some form of SQL encoding for storage, I was curious if it could store the world chunks in an SQL server

**Comment by yamayaki1** [comment]
No, Cesium can not do that. It is based on a key-value database (NoSQL/ LMDB).
